curation_concerns-models/app/actors/curation_concerns/file_set_actor.rb

```diff
@@ -50,14 +50,14 @@ def create_metadata(upload_set_id, work, file_set_params = {})
     # Simultaneously moving a preservation copy to the repostiory.
     # TODO: create a job to monitor this directory and prune old files that
     # have made it to the repo
-    # @param [ActionDigest::HTTP::UploadedFile, Tempfile] file the file uploaded by the user.
+    # @param [File, ActionDigest::HTTP::UploadedFile, Tempfile] file the file uploaded by the user.
     def create_content(file)
-      file_set.label ||= file.original_filename
-      file_set.title = [file_set.label] if file_set.title.blank?
```

Comment:
Doing `file_set.title = [file_set.label] if file_set.title.blank?` here satisfies the tests.  Putting the same line in the IngestFileJob does not persist the title.